portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,111 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+ //Create a boolean flag to note when the button for random show and random fact have been 
+ //clicked so they will open on first click and close on the second click
+showHasBeenClicked = false;
+factHasBeenClicked = false;
+
+function recommendRandomShow() {
+    //options for randomiser
+  const shows =
+    ['The Good Place', 'Gossip Girl', 'Community', 'Emily in Paris', 'How I met Your Mother'];
+    
+  // Pick a random tv show.
+  const show = shows[Math.floor(Math.random() * shows.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const showContainer = document.getElementById('tv-show-container');
+  //check if the button has been clicked already and if yes collapse container
```

Comment:
nit: comments should start with a space, then a capital letter and end with a full stop, so they read like a sentence. See comment above for example: // Add it to the page.
Could you fix up the comments in this file.

Reply:
Fixed.